client/client-multi/client-c/src/kaa/gen/kaa_configuration_gen.c

```diff
@@ -93,7 +93,7 @@ static void kaa_union_null_or_fixed_serialize(avro_writer_t writer, void *data)
         }
     }
 }
-static kaa_union_t *kaa_union_null_or_fixed_create()
+static kaa_union_t *kaa_union_null_or_fixed_create(void)
```

Comment:
The file is auto-generated, so to apply these changes source templates should be edited as well.
Please, see 
https://github.com/kaaproject/kaa/tree/master/avrogenc/src/main/resources
https://github.com/kaaproject/kaa/tree/master/server/control/src/main/resources/sdk/c